### PR TITLE
Adding Marketplace module

### PR DESCRIPTION
**Pull request by olrakovs-zz**
## Description
 

 General availability of `Az.Marketplace` module. With  adding Privatestore cmdlets
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by olrakovs** [comment]
Related design review "Azure Marketplace Adding Privatestore cmdlets" #608

**Comment by VeryEarly** [comment]
Az.Marketplace cannot be GA this time, please read the GA process:
https://github.com/Azure/azure-powershell-pr/wiki/Process-of-GA-module

**Comment by olrakovs** [comment]
@VeryEarly  thank you for your comment , committed some changes for preview

File: src/Marketplace/Marketplace/Az.Marketplace.psd1 [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+DotNetFrameworkVersion = '4.7.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.7.5'; })
```
Comment (VeryEarly) [inline src/Marketplace/Marketplace/Az.Marketplace.psd1:56]: Is it possible to reference to the latest Az.Accounts-1.8.1?

File: src/Marketplace/Marketplace/ChangeLog.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Upcoming Release
+
+## Version 1.0.0-preview
```
Comment (VeryEarly) [inline src/Marketplace/Marketplace/ChangeLog.md:22]: please change this version to 0.1.0 (without '-preview')

**Comment by olrakovs** [comment]
Hey @VeryEarly who need to merge this pr?

**Comment by VeryEarly** [comment]
> Hey @VeryEarly who need to merge this pr?

Done

**Pull request merged**